app/views/team_members/show.html.haml

```diff
@@ -3,6 +3,8 @@
   .spotlight-holder.py-lg.pt-lg-xl
     .container.d-flex.align-items-center.no-padding
       .col
+        = link_to team_members_path, class: "btn btn-md border-white text-white bg-dark" do
+          Back to team
```

Comment:
What would you think of capitalizing the `t` in `team`?

Reply:
I was reading a style guide that was talking about when to titleize but that probably shouldn't apply to buttons. Thanks for the pointer!